src/utils/helpers/parseArtifactList.ts

```diff
@@ -15,8 +15,8 @@ const isPathAbsolute = (path: string) => {
 
 export const artifactBaseUrl = (apiUrl: string, baseUrl: string) => {
   if (isPathAbsolute(apiUrl)) {
-    return apiUrl;
+    return `${apiUrl}/`;
```

Comment:
It there a better more typescript-y way of doing this?

Reply:
Not sure where the appropriate place to write a test is, would appreciate some direction :pray: